providers/implementations/asymciphers/rsa_enc.c

```diff
@@ -158,6 +158,19 @@ static int rsa_encrypt(void *vprsactx, unsigned char *out, size_t *outlen,
     if (!ossl_prov_is_running())
         return 0;
 
+#ifdef FIPS_MODULE
+    if ((prsactx->pad_mode == RSA_PKCS1_PADDING
+         || prsactx->pad_mode == RSA_PKCS1_WITH_TLS_PADDING)
+        && !ossl_FIPS_IND_on_unapproved(OSSL_FIPS_IND_GET(prsactx),
```

Comment:
any reason to not use the macro here?

Reply:
Copied from somewhere else I think.  Worth fixing in due course.

Reply:
FYI - It only does this in a few places in order to share the code